### ISSUE TITLE
general setup for COMports across tasks

### ISSUE THREAD

**Issue by anne-urai** [int-brain-lab/iblrig#67]
allow the user to setup all comports only once in a single setup file, instead of having to go through all task files and listing the right comports (will be especially useful when adding new tasks)

**Comment by nbonacchi** [comment]
Yes, I've delayed doing this too long and I finally did it during the week end, it should be in the next release, which will be 3.5.0. I'm just confirming that the update will work and not brake any of the tasks...
